Package.swift

```diff
@@ -45,25 +45,19 @@ let package = Package(
 
 var swiftSettings: [SwiftSetting] {
     [
-        .enableUpcomingFeature("ExistentialAny"),
-        .enableUpcomingFeature("ConciseMagicFile"),
-        .enableUpcomingFeature("ForwardTrailingClosures"),
-        .enableUpcomingFeature("DisableOutwardActorInference"),
-        .enableUpcomingFeature("StrictConcurrency"),
-        .enableExperimentalFeature("StrictConcurrency=complete"),
-        .enableExperimentalFeature("AccessLevelOnImport"),
+        .enableUpcomingFeature("ExistentialAny")
     ]
 }
 
 if let target = package.targets.filter({ $0.name == "CMinizip" }).first {
     #if os(Windows)
-        if ProcessInfo.processInfo.environment["ZIP_USE_DYNAMIC_ZLIB"] == nil {
-            target.cSettings?.append(contentsOf: [.define("ZLIB_STATIC")])
-            target.linkerSettings = [.linkedLibrary("zlibstatic")]
-        } else {
-            target.linkerSettings = [.linkedLibrary("zlib")]
-        }
+    if ProcessInfo.processInfo.environment["ZIP_USE_DYNAMIC_ZLIB"] == nil {
```

Comment:
```suggestion
    if Context.environment["ZIP_USE_DYNAMIC_ZLIB"] == nil {
```
Using SwiftPM's provided `Context` global allows you to avoid having to import `Foundation` in your manifest.